pkg/rtc/utils.go

```diff
@@ -126,7 +123,9 @@ func Recover(l logger.Logger) {
 		default:
 			err = errors.New("unknown panic")
 		}
-		l.Errorw("recovered panic", err, "panic", r)
+		l.Errorw("Panic", err, "panic", r)
```

Comment:
use lower case `panic`, we use lower case log lines, probably some places have upper case, but we should standardize